File: distint-languaje/AreaTriangulo.java
import java.util.Scanner;

class AreaTriangulo{
static Scanner teclado=new Scanner(System.in);
public static void main(String [] arg){
  //Definir variables y otros
  System.out.println("Hola Mundoo");
  int b, h, area;
  //Datos de entrada
  System.out.println("Ingrese la Base");
  b=teclado.nextInt();
  System.out.println("Ingrese la Altura");
  h=teclado.nextInt();
  //Proceso
  area=(b*h)/2;
  //Datos de salida
  System.out.println("Area de un Triangulo es:"+area);

}
}

File: distint-languaje/AreaTriangulo.kt
fun main() {
  //Definir Variables y otros
   println("Ejercicio 01:Area de un Triangulo")
   var b: Int?
   var h: Int?
   var area: Int?
   //Datos de entrada
   println("Ingrese Base:")
   b=readLine()?.toIntOrNull()!!
   println("Ingrese Altura:")
   h=readLine()?.toIntOrNull()!!
   //Proceso
   area=(b*h)/2
   //Datos de salida
   println("El area del triangulo es:"+area)


}

   
